core-tests/jvm/src/test/scala/zio/internal/FiberInterruptSpec.scala

```diff
@@ -0,0 +1,38 @@
+package zio.internal
+
+import zio.Promise.internal.Pending
+import zio.duration._
+import zio.test.Assertion.equalTo
+import zio.test.TestAspect._
+import zio.test.environment.Live
+import zio.test.{ ZSpec, assertM, suite, testM }
+import zio.{ Promise, ZIOBaseSpec }
+
+object FiberInterruptSpec extends ZIOBaseSpec {
+
+  override def spec: ZSpec[Environment, Failure] =
```

Comment:
Was this testing failing for you before? It is consistently passing for me on `master` even without the `nonFlaky`.